### DISCUSSION TITLE
KnowledgeSystem/FrontEnd/HTML/html5Tag

### DISCUSSION THREAD

**Discussion by giscus** [rockbenben/LearnData#47] [Comments]
# KnowledgeSystem/FrontEnd/HTML/html5Tag

进击的 HTML HTML5 给开发者提供了哪些便利呢？简单列举有： 用于绘画的 canvas 元素 用于媒介播放的 video 和 audio 元素 对本地离线存储更好的支持（localStorage、sessionStorage） 新的语义化标签（article、footer、header、nav、section…) 新的表单控件（calendar...

https://www.zxwin0125.top/KnowledgeSystem/FrontEnd/HTML/html5Tag.html

**Comment by zxwin0125** [comment]
已完成整理
